src/main/java/EPparser/CommandParser.java

```diff
@@ -0,0 +1,237 @@
+package EPparser;
+
+import commands.*;
```

Comment:
Avoid usage of wildcard imports.

![image](https://user-images.githubusercontent.com/35756209/67217743-d2f34080-f457-11e9-834d-3669db2886ea.png)